components/InformationBoxes.jsx

```diff
@@ -15,6 +15,7 @@ const defaultProps = {
         amount: PropTypes.number,
         isMonetary: PropTypes.bool,
         subtitle: PropTypes.string,
+        countSubtitle: PropTypes.string,
```

Comment:
The only thing I don't like is the name "countSubtitle", it implies this will always be some kind of count. But I would also object to "amount" for the same reason. Should we take the opportunity to make these more generic so we don't have to if another case comes up? And should the storybook have an example or two of this new feature?

Reply:
@brett-varney I agree that countSubtitle should be more generic since the a/c describes this as an optional 4th line of text - what do you think about the name `dataSubtitle` or `dataNote` ?

Thanks - will update Storybook with examples!

Reply:
> 
> 
> @brett-varney I agree that countSubtitle should be more generic since the a/c describes this as an optional 4th line of text - what do you think about the name `dataSubtitle` or `dataNote` ? or `subtitleBottom` to be more generic?
> 
> Thanks - will update Storybook with examples!

I guess subtitleBottom makes the most sense for now, but someday they'll probably all need to be updated.